skin-schema.json

```diff
@@ -689,6 +689,17 @@
               "default": "false"
             }
           }
+        },
+        "qualitySelection":{
+          "description":"Configuration for the control bar quality selection panel/popover",
+          "type":"object",
+          "properties":{
+            "format":{
+              "description":"Determine the format of the quality options. Available formats: bitrate only, resolution only, and resolution + bitrate. Comma or white space separated values of: \'bitrate\' and \'resolution\'",
+              "type":"string",
```

Comment:
I think it could be useful to add `enum` here in order to limit the setting to valid values.

Reply:
Added